pkg/cmd/release/view/view.go

```diff
@@ -67,7 +67,7 @@ func NewCmdView(f *cmdutil.Factory, runF func(*ViewOptions) error) *cobra.Comman
 	}
 
 	cmd.Flags().BoolVarP(&opts.WebMode, "web", "w", false, "Open the release in the browser")
-	cmdutil.AddJSONFlags(cmd, &opts.Exporter, shared.ReleaseFields)
+	cmdutil.AddJSONFlags(cmd, &opts.Exporter, api.ReleaseFields)
```

Comment:
The biggest issue I'm encountering right now is that regardless of the fields that I pass to `--json`, the output is the entire release object. I noticed that the `pr` command manually constructs the graphql query via strings and filtering on the provided fields. However, that didn't feel like the preferred way to go, but I'm unsure how to modify the query to limit the fields. Any pointers would be appreciated!